mlir/lib/Target/LLVMIR/ModuleImport.cpp

```diff
@@ -1610,6 +1611,16 @@ void ModuleImport::processFunctionAttributes(llvm::Function *func,
     funcOp.setArmStreaming(true);
   else if (func->hasFnAttribute("aarch64_pstate_sm_body"))
     funcOp.setArmLocallyStreaming(true);
+  llvm::Attribute attr = func->getFnAttribute(llvm::Attribute::VScaleRange);
+  if (attr.isValid()) {
+    MLIRContext *context = funcOp.getContext();
+    auto intTy = IntegerType::get(context, 32);
+    std::optional<unsigned> maxAttr = attr.getVScaleRangeMax();
+    unsigned maxVal = maxAttr.value_or(0);
```

Comment:
```suggestion
    unsigned maxVal = attr.getVScaleRangeMax().value_or(0);
```

Reply:
I went one step further, and stuck the whole `attr.getVscaleRangeMax().value_or(0)` into the place using maxVal.